src/common/components/repositories-list/index.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import conf from '../../helpers/config';
+import { createSnap } from '../../actions/repository-input';
+import { Message } from '../forms';
+import RepositoryRow from '../repository-row';
+import Spinner from '../spinner';
+
+// loading container styles not to duplicate .spinner class
+import styles from '../../containers/container.css';
+
+class RepositoriesList extends Component {
+
+
+  getErrorMessage() {
+    const input = this.props.repositoryInput;
+
+    if (input.error) {
+      const payload = input.error.json.payload;
+      if (payload.code === 'snap-name-not-registered') {
+        const snapName = payload.snap_name;
+        const devportalUrl = conf.get('STORE_DEVPORTAL_URL');
+        const registerNameUrl = `${devportalUrl}/click-apps/register-name/` +
+                                `?name=${encodeURIComponent(snapName)}`;
+        return (
+          <span>
+            The name provided in the snapcraft.yaml file ({snapName}) is not
+            registered in the store.
```

Comment:
"the store" is probably too vague, linking the term with the url would put it beyond doubt.